docs/serverless-v2/08-01-expose-lambda.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Expose lambdas with API Rules
+type: Tutorials
+---
+
+This tutorial shows how you can expose a lambda function to access it outside the cluster, through an HTTP proxy. To expose it, use an APIRule custom resource (CR) managed by the in-house API Gateway Controller. This controller reacts to an instance of the APIRule CR and, based on its details, it creates an Istio Virtual Service and Oathkeeper Access Rules that specify your permissions for the exposed function.
+
+This example is based on a simple code which returns the "Hello World!" string when you access the HTTP host on which the lambda is exposed. This lambda is:
+
+- Not secured by an access token (**handler** set to `noop` in the APIRule CR).
+- Accepts `GET`, `POST`, `PUT`, and `DELETE` request types.
+
+>**NOTE:** To learn more about securing your lambda, see [this](/components/api-gateway-v2/#tutorials-expose-and-secure-a-service-deploy-expose-and-secure-the-sample-resources) tutorial.
+
+## Steps
+
+Follows these steps:
+
+1. Export these variables:
+
+    ```bash
+    export DOMAIN={DOMAIN_NAME}
+    export NAME={RESOURCE_NAME}
+    export NAMESPACE="serverless"
+    ```
+    >**NOTE:** Lambda takes the name from the Function CR name. The APIRule CR can have a different name but for the purpose of this tutorial, all related resources share a common name defined under the **NAME** variable.
+
+2. Create a Function CR that specifies the lambda's logic and defines a runtime on which it should run:
```

Comment:
Hm... I think we should have 2 tutorials:
1. How to create a lambda
2. How to expose a lambda (without lambda creation)

Reply:
yep, I will divide it into two separate tutorials

Reply:
Yes, we should and yes, we will ;) 👍